packages/simcore-sdk/src/simcore_sdk/node_ports_common/exceptions.py

```diff
@@ -72,6 +72,13 @@ def __init__(self, msg: str | None = None):
         super().__init__(msg or "Error while transferring to/from S3 storage")
 
 
+class AWSS3400RequestTimeOutError(NodeportsException):

```

Comment:
MINOR: just to make it more readable, why not to `AwsS3RequestTimeoutError`

Reply:
Q: [400](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400) is bad request error, right?  And the reason must be "timeout", is that right?
If that is the case, do you map to this error just when you get 400 or "400 because of timeout"?


A: OK, this piece of code below answers it. 
```python
        if e.status == web.HTTPBadRequest.status_code and "RequestTimeout" in e.body:
            raise exceptions.AWSS3400RequestTimeOutError(e.body) from e
```

Reply:
> MINOR: just to make it more readable, why not to `AwsS3RequestTimeoutError`

At this point I would go for `AwsS3BadRequestRequestTimeoutError` at least it references the issue.

Reply:
`RequestRequest` ? :-) aren't we overdoing it a bit with names? Anyway, if you really like this name, just go ahead